deps/build.jl

```diff
@@ -6,6 +6,8 @@ using BinDeps
 
 Sys.WORD_SIZE != 64 && error("Build script configured only for x64")
 
+download_dir = joinpath(@__DIR__, "downloads")
```

Comment:
Only used for OSX build. Maybe move there? Also are you sure you'll have `@__DIR__` pointing somewhere sensible when you probe it?